run_ui.sh

```diff
@@ -1,3 +1,5 @@
+
+
 source "$(poetry env info --path)/bin/activate"
 python scripts/tracking.py skyvern-oss-run-ui
```

Comment:
Why not move the tracking into `streamlit_app/visualizer/streamlit.py`?

Reply:
This PR has nothing to do with tracking